sdk/mongocluster/Azure.ResourceManager.MongoCluster/samples/Generated/Samples/Sample_MongoClusterCollection.cs

```diff
@@ -22,7 +22,7 @@ public partial class Sample_MongoClusterCollection
         public async Task CreateOrUpdate_CreatesANewMongoClusterResource()
         {
             // Generated from example definition: 2024-07-01/MongoClusters_Create.json
-            // this example is just showing the usage of "MongoCluster_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "MongoClusters_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `createOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/mongocluster/DocumentDB.MongoCluster.Management/MongoCluster.tsp#L32